.github/ISSUE_TEMPLATE/request-feature.yml

```diff
@@ -7,9 +7,9 @@ assignees:
   - EdVraz
   - Astrea49
   - Toricane
+  - Catalyst4222
 body:
   - type: markdown
-    id: thanks
```

Comment:
Why is the ID removed here? (and in report-bug.yml as well)

Reply:
lol ive asked the same, see above